docker-compose.yml

```diff
@@ -250,7 +250,7 @@ services:
     command: bash -c "composer update && php restart.php"
 
   dynamo:
-    image: amazon/dynamodb-local
+    image: amazon/dynamodb-local:latest
```

Comment:
Same question as above

Reply:
They don't provide major or minor tags, only patch-level. I debated, but thought we probably wouldn't want to keep up with patch-level updates.